packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -359,3 +368,57 @@ export type GPCRevealedClaims = {
    */
   watermark?: PODValue;
 };
+
+/**
+ * General-purpose circuit requirements for a given proof configuration.
+ * These values will be appropriately checked against the circuit configurations
+ * at our disposal in order to accommodate all of the proof inputs. For the
+ * objects, entries, Merkle tree depth and membership list sizes, this amounts
+ * to picking the circuit description whose corresponding parameters exceed
+ * these numbers, while the choice of tuple parameters is more involved.
+ */
+export type GPCRequirements = {
```

Comment:
This doesn't currently need to be exported from the package (which it will be by default via its inclusion here).  I'm not sure it's a bad thing to expose and explain this concept, but right now it's never exposed for users outside the package.  This type could go in gpcUtil to maintain that, though I'm ambivalent.